app/frontend/src/reactQueryClient.tsx

```diff
@@ -11,7 +11,13 @@ import { reactQueryRetries } from "./constants";
 
 export const queryClient = new QueryClient({
   //grpc-web has built in timeout, so better not use the default exponential backoff
-  defaultOptions: { queries: { retry: reactQueryRetries, retryDelay: 0 } },
+  defaultOptions: {
+    queries: {
+      refetchOnWindowFocus: false,
```

Comment:
Refetching every time you re-focus the browser window seems a bit too much to me, so I thought might be worth disabling.